streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreChangelogReader.java

```diff
@@ -236,7 +235,7 @@ public StoreChangelogReader(final Time time,
         this.stateRestoreListener = stateRestoreListener;
         this.standbyUpdateListener = standbyUpdateListener;
 
-        this.stateUpdaterEnabled = InternalConfig.stateUpdaterEnabled(config.originals());
```

Comment:
I think we only use this in one place, and it should be possible to remove the variable (and it's usage) completely, in one go